awx_collection/plugins/module_utils/awxkit.py

```diff
@@ -42,7 +43,16 @@ def get_api_v2_object(self):
         if not self.apiV2Ref:
             if not self.authenticated:
                 self.authenticate()
-            v2_index = get_registered_page('/api/v2/')(self.connection).get()
+            # Since we are using awxkit, start with AWXKIT_API_BASE_PATH if set
+            prefix = getenv('AWXKIT_API_BASE_PATH', '/api/')
```

Comment:
we can remove this, awxkit should instead honor CONTROLLER_OPTIONAL_API_URLPATTERN_PREFIX. See https://github.com/ansible/awx/pull/16085